packages/components/addon/components/hds/accordion/item/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { guidFor } from '@ember/object/internals';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class HdsAccordionItemIndexComponent extends Component {
+  @tracked isOpen = this.args.isOpen ?? false;
+
+  /**
+   * Generates a unique ID for the Content
+   *
+   * @param contentId
+   */
+  contentId = 'content-' + guidFor(this);
+
+  /**
+   * @param ariaLabel
+   * @type {string}
+   * @default 'Toggle display'
+   */
+  get ariaLabel() {
+    return this.args.ariaLabel ?? 'Toggle display';
+  }
+
+  /**
+   * @param isClickable
+   * @type {boolean}
+   * @default true
+   */
+  get isClickable() {
```

Comment:
`isInteractive` may be more broad (not to tie it to a mouse interaction) – otherwise happy to stick to `isClickable` as it aligns with the existing `onClickToggle`
```suggestion
  get isClickable() {
```

Reply:
+1 to `isInteractive` 
but maybe we can also include a hint that it's fullWidth? 

`isFullWidthInteractive` or something similar?

Reply:
The "interactive" wording is good I think. I can use that.

You don't think `isFullWidthInteractive` is a bit long?

Maybe I'll try changing it to just `isInteractive` to start. Then if that still doesn't seem clear enough I can change it again before release.

Reply:
I will _always_ vote for more clarity over terseness, heh. Totally fine with whatever we decide though!

Reply:
Ok well I updated it to `isInteractive` for now. We can think about it a bit more and change again before release if folks don't think it's clear enough.

Reply:
"expandInteractiveArea" is similar to how I usually describe the technique I'm doing with the button so that could be good. Anyway I'll think about it more and we can wait to make a decision.

Reply:
I'm now leaning toward either `isFullWidthInteractive` or `isFullAreaInteractive` but still open to other suggestions. Maybe we should take a vote?

@MelSumner @alex-ju @didoo @cveigt

Reply:
Another naming option to consider:

`isNestedInteractive` Default would be `false`

Reply:
Thanks for weighing in on naming. Going to go with `containsInteractive` from Ambassador votes and our overall team feedback.